.github/workflows/ci.yml

```diff
@@ -50,3 +50,13 @@ jobs:
       flink_version: ${{ matrix.flink }}
       timeout_global: 120
       timeout_test: 80
+  CheckDeadLinks:
```

Comment:
Please revert this change and do it in another patch. We may review #78 first. Pleaase avoid squash commits in cherry-pick.

Reply:
@tisonkun I just check this branch with no dead link. I had drop this commit.